snmp/tests/compose/data/meraki-cloud-controller.snmprec

```diff
@@ -1,44 +1,23 @@
 1.3.6.1.2.1.1.1.0|4|Cisco Meraki Cloud Controller
 1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.29671.1
-1.3.6.1.2.1.1.3.0|67|3046294100
 1.3.6.1.2.1.1.4.0|4|support@meraki.com
 1.3.6.1.2.1.1.5.0|4|dashboard.meraki.com
 1.3.6.1.2.1.1.6.0|4|123 Fake Blvd, San Francisco, CA 94158, USA
+1.3.6.1.2.1.2.1.0|2|28
 1.3.6.1.2.1.2.2.1.1.11|2|11
 1.3.6.1.2.1.2.2.1.2.11|4x|42010aa40033
 1.3.6.1.2.1.2.2.1.3.11|2|29
 1.3.6.1.2.1.2.2.1.4.11|2|10
-1.3.6.1.2.1.2.2.1.5.11|66|842689539
 1.3.6.1.2.1.2.2.1.6.11|4x|00
-1.3.6.1.2.1.2.2.1.7.11|2|2
-1.3.6.1.2.1.2.2.1.8.11|2|7
 1.3.6.1.2.1.2.2.1.9.11|67|4043941980
 1.3.6.1.2.1.2.2.1.10.11|65|2268799121
 1.3.6.1.2.1.2.2.1.11.11|65|1358081555
 1.3.6.1.2.1.2.2.1.12.11|65|2804916297
-1.3.6.1.2.1.2.2.1.13.11|65|1953269825
-1.3.6.1.2.1.2.2.1.14.11|65|3659465164
```

Comment:
I think removing this might be the source of the error shown. Some or most of these might still be needed.